Fix xP0610L and BS250

evanshultz opened the pull request:
Fixes https://github.com/KiCad/kicad-symbols/issues/2041

- TP0610L and VP0610L were an ALIAS of 2N7000 but they are P-channel; make them an ALIAS of BS250 (same datasheet)
- Sadly, BS250 had the S and D pins swapped

https://www.vishay.com/docs/70209/70209.pdf

![image](https://user-images.githubusercontent.com/1936989/64656259-c359f400-d3e3-11e9-9305-b3d819be5ef7.png)

---

All contributions to the kicad library must follow the [KiCad library convention](http://kicad-pcb.org/libraries/klc/)

Thanks for creating a pull request to contribute to the KiCad libraries! To speed up integration of your PR, please check the following items:

- [ ] Provide a URL to a datasheet for the symbol(s) you are contributing
- [ ] An example screenshot image is very helpful
- [ ] Ensure that the associated footprints match the [official footprint library](https://github.com/kicad/kicad-footprints)
  - A new fitting footprint must be submitted if the library does not yet contain one.
- [ ] If there are matching footprint PRs, provide link(s) as appropriate
- [ ] Check the output of the Travis automated check scripts - fix any errors as required
- [ ] Give a reason behind any intentional library convention rule violation.

cpresser commented:
BS250 is not the same as TP0610L and VP0610L. They are in the same datasheet, but have a different pinout. So they cant be an alias.
![image](https://user-images.githubusercontent.com/17516572/64739563-bed03100-d4f2-11e9-9d7b-2427e08ca4d6.png)

* BS250 has DGS
* TP0610L and VP0610L have SGD

TO-92 should work for both types, TO-29L in the library seems to same as TO-226

evanshultz commented:
Yesterday was not my day for making changes...

Thanks for catching and noticing my error. Fixed now. I also corrected BS250's drain-source voltage.
